src/apps/src/main/scala/osmesa/apps/streaming/AOIMonitor.scala

```diff
@@ -20,15 +20,15 @@ import org.locationtech.jts.{geom => jts}
 import org.locationtech.jts.geom.prep._
 import org.locationtech.jts.io.WKBReader
 import osmesa.analytics.Analytics
-import osmesa.analytics.oneoffs.Interval._
 import osmesa.analytics.stats._
+import osmesa.apps.streaming.Interval._
```

Comment:
If `Interval` has a broad application, I suggest maintaining it in the `analytics` package so that when we publish that package as a library, it will be available more broadly (for example, when this `AOIMonitor` class moves out to the SotD repo.

Reply:
Nope. It currently doesn't. It's defined in this file and will move with the code when we move this command elsewhere since its just a duplicate of an object that SotD already defines. Just going to leave as is for now.